### DISCUSSION TITLE
What's the license of SE documentation?

### DISCUSSION THREAD

**Discussion by zcraber** [SubtitleEdit/subtitleedit#6545] [General]
Asking this here as I couldn't find the license info on the [webpage](https://www.nikse.dk/subtitleedit/help). Is it allowed to copy/modify/redistribute?

**Comment by niksedk** [comment]
It's here: https://github.com/SubtitleEdit/subtitleedit/blob/main/LICENSE.txt

You're allowed to copy/modify/redistribute - but modifications must be public.

**Reply by zcraber** [reply]
Oh ok, I thought that license is meant only for the software source code.
